client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/address-step/index.js

```diff
@@ -56,7 +56,7 @@ const renderSummary = (
 	}
 	str += 'US' === country ? postcode.split( '-' )[ 0 ] : postcode;
 	if ( showCountry ) {
-		str += ', ' + countriesData[ country ].name;
+		str += ', ' + ( countriesData[ country ] ? countriesData[ country ].name : country );
```

Comment:
Is now a good time to rename `showCountryInSummary` to `shouldShowCountryInSummary`? Happy to see it happen in a subsequent PR too.

Reply:
I've just changed it in https://github.com/Automattic/wp-calypso/pull/25723/commits/44bc8529dd8d9ad87948a8c8291a9fee01126e92. In the `/continents` PR this file has changed a lot, so it made sense to change it there.